src/resources/language/_language-ja.yml

```diff
@@ -70,16 +70,16 @@ environment-solution-title: "解答"
 
 listing-page-order-by: "並び替え"
 listing-page-order-by-default: "デフォルト"
-listing-page-order-by-date-asc: "日付の昇順"
-listing-page-order-by-date-desc: "日付の降順"
-listing-page-order-by-number-desc: "ページ番号の降順"
-listing-page-order-by-number-asc: "ページ番号の昇順"
+listing-page-order-by-date-asc: "日付（昇順）"
+listing-page-order-by-date-desc: "日付（降順）"
+listing-page-order-by-number-desc: "ページ番号（昇順）"
+listing-page-order-by-number-asc: "ページ番号（昇順）"
 listing-page-field-date: "日付"
 listing-page-field-title: "題名"
 listing-page-field-description: "説明"
 listing-page-field-author: "作者"
 listing-page-field-filename: "ファイル名"
-listing-page-field-filemodified: "ファイル変更"
+listing-page-field-filemodified: "更新"
```

Comment:
I would suggest `更新日`.
In English, `Modified` is preferred maybe because it is fairly shorter than `Modified data`.
OTH, adding a letter in Japanese is small enough.
Note that Spanish version is more like `更新日` (`Fecha de modificación`).

Reply:
Thanks for your suggestion!
That make sense, this seems a "date" type.

https://github.com/quarto-dev/quarto-cli/blob/e8dd7e1448cbce1c52c04274222a8e731780bc6c/src/project/types/website/listing/website-listing-read.ts#L127